client/src/shared/components/ProgressBar/ProgressBar.styled.ts

```diff
@@ -0,0 +1,31 @@
+import styled from '@emotion/styled';
+
+export type StyledProgressProps = {
+  height: string;
+  backgroundColor: string;
+  borderRadius: string;
+};
+
+export type StyledFillProps = {
+  width: number;
+  color: string;
+  borderRadius: string;
+  animated: boolean;
+};
+
+export const ProgressContainer = styled.div<StyledProgressProps>`
```

Comment:
p2.
다음과 같이 지정해두면, 기존에 있는 타입에서 확장가능하게 사용 가능할 것 같은데요! 따로 타입을 정의해두신 이유가 있으신가요?? 
```suggestion
export const ProgressContainer = styled.div<Pick<ProgressBarProps, 'height' | 'backgroundColor' | 'borderRadius'>>`
```

Reply:
p1.
세라의 의견에 동의합니다!
기존에 있던 타입을 최대한 활용하며 재사용하는 것이 좋아보여요!!